go.sum

```diff
@@ -1,33 +1,153 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
+cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
+foxygo.at/protog v0.0.14 h1:A8wnusX0iDVDqfuD8XXQxTJyZvqD0lwApQf2L29uqGc=
+foxygo.at/protog v0.0.14/go.mod h1:z0MaC+G1OT/hhL1rstKjhqCGT7AeyG0kmWxZQDwIh8w=
+github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
 github.com/alecthomas/kong v0.4.1 h1:0sFnMts+ijOiFuSHsMB9MlDi3NGINBkx9KIw1/gcuDw=
 github.com/alecthomas/kong v0.4.1/go.mod h1:uzxf/HUh0tj43x1AyJROl3JT7SgsZ5m+icOv1csRhc0=
 github.com/alecthomas/participle/v2 v2.0.0-alpha7 h1:cK4vjj0VSgb3lN1nuKA5F7dw+1s1pWBe5bx7nNCnN+c=
 github.com/alecthomas/participle/v2 v2.0.0-alpha7/go.mod h1:NumScqsC42o9x+dGj8/YqsIfhrIQjFEOFovxotbBirA=
 github.com/alecthomas/repr v0.0.0-20181024024818-d37bc2a10ba1/go.mod h1:xTS7Pm1pD1mvyM075QCDSRqH6qRLXylzS24ZTpRiSzQ=
 github.com/alecthomas/repr v0.0.0-20210801044451-80ca428c5142 h1:8Uy0oSf5co/NZXje7U1z8Mpep++QJOldL2hs/sBQf48=
 github.com/alecthomas/repr v0.0.0-20210801044451-80ca428c5142/go.mod h1:2kn6fqh/zIyPLmm3ugklbEi5hg5wS435eygvNfaDQL8=
+github.com/antihax/optional v1.0.0/go.mod h1:uupD/76wgC+ih3iEmQUL+0Ugr19nfwCT1kdvxnR2qWY=
+github.com/census-instrumentation/opencensus-proto v0.2.1/go.mod h1:f6KPmirojxKA12rnyqOA5BBL4O983OfeGPqjHWSTneU=
```

Comment:
This is pulling in a lot of new deps, what are these from do you know?

Reply:
It is a shame to see all this extra mess, but I don't think they are deps. They come from the protog module as it has other code in it (grpc) that this go.sum stuff must come from. The protog/registry stuff only depends on google.golang.org/protobuf, so besides the protog/registry dependency, I don't think there are any extra real deps.

As mentioned above, if we can resolve the options earlier and not go via prototext, we won't need this dep at all. I understand the desire to keep the deps low, so I'll see how much work that will be.

Reply:
This has gone now. Instead I've put a stripped-down version of the protog registry with the compiler.